Userland/Libraries/LibUnicode/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 include(${SerenityOS_SOURCE_DIR}/Meta/CMake/unicode_data.cmake)
 
+SET(SOURCES ${UNICODE_DATA_SOURCES})
```

Comment:
Not sure what happens when Unicode data sources are empty, should we add some conditional logic for that? 

Also pre-existing, these SET are a bit screamy (also CMake 2.8-y), might uncaps them for consistency with other files.

Reply:
Or rather. Should we set ENABLE_UNICODE_DATA as a #define for this library too? And what happens when it's false?

Reply:
Fixed!

Reply:
Oh, oops for some reason only your first comment showed up when I first looked at this.

When ENABLE_UNICODE_DATA is off, CMake complained about this new shared library being empty. So I conditionalized its creation/linkage against UNICODE_DATA_SOURCES being defined, since that only happens when ENABLE_UNICODE_DATA is on.